pandas/io/json/normalize.py

```diff
@@ -149,6 +175,20 @@ def json_normalize(data, record_path=None, meta=None,
     1  NaN         NaN      Regner        NaN       Mose       NaN
     2  2.0  Faye Raker         NaN        NaN        NaN       NaN
 
+    >>> from pandas.io.json import json_normalize
+    >>> data = [{'id': 1,
+    ...          'name': {'first': 'Coleen', 'last': 'Volk'},
+    ...          "fitness": {"height":130, "weight":60}},
+    ...         {'name': {'given': 'Mose', 'family': 'Regner'},
```

Comment:
this won't pass doc-strings, I guess its not being checked? also you have mixed quoting, is this an actual result?

@WillAyd